lib/challenge.js

```diff
@@ -60,6 +111,12 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function(name, species, breed, noise){
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+}
```

Comment:
So far, so good. We were looking for you to put a prototype method on Pet that uses `this` to reference the object and access its `noise` property. Partial credit.